src/src/components/form/CustomSection.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <th colspan="4" class="sectionHeader">
```

Comment:
Codacy has a fix for the issue: ['class' should be on a new line.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
  <th colspan="4"
```